demisto_sdk/commands/validate/validators/RN_validators/RN114_validate_release_notes_header.py

```diff
@@ -116,9 +117,10 @@ def extract_rn_headers(self, release_note_content: str) -> Dict[str, List[str]]:
                     header = (
                         content_type_section[0].rstrip()
                         if ("New: " not in content_type_section[0])
-                        else content_type_section[0][len("New: ") :]
+                        else content_type_section[0].removeprefix("New: ")
                     )
-                    headers.setdefault(content_type, []).append(header)
+                    if header not in PB_RELEASE_NOTES_FORMAT:  # remove PB custom headers
+                        headers.setdefault(content_type, []).append(header)
```

Comment:
Let's move the if section outside of this function scope as the main purpose is to extract the headers and not to filter them. Please open a new function that will filter the headers and add the filters we talked about in the `validate_release_notes_headers`